src/wallet/rpc/wallet.cpp

```diff
@@ -701,6 +701,36 @@ RPCHelpMan simulaterawtransaction()
     };
 }
 
+static RPCHelpMan migratewallet()
+{
+    return RPCHelpMan{"migratewallet",
+        "\nEXPERIMENTAL warning: This call may not work as expected and may be changed in future releases\n"
+        "\nMigrate the wallet to a descriptor wallet.\n"
+        "A new wallet backup will need to be made.",
```

Comment:
Maybe this could mention that a backup will be created in the same manner as by using `backupwallet`, that can be restored using `restorewallet`?

Reply:
Added some text about that.